README.md

```diff
@@ -9,3 +9,16 @@
 `rails db:migrate`
 
 `rails hydra:server`
+
+# Quickstart development with Docker
+Docker provides an alternative way of setting up an Avalon Media System Development Environment in minutes without installing any dependencies beside Docker itself. It should be noted that the docker-compose.yml provided here is for development only and will be updated continually.
+* Install [Docker](https://docs.docker.com/engine/installation/) and [docker-compose](https://docs.docker.com/compose/install/)
+* ```git clone https://github.com/avalonmediasystem/tbd```
+* ```cd tbd```
+* ```docker-compose up```
+* Try loading Avalon in your browser: ```localhost:3000```
+
+Avalon is served by Webrick in development mode so any changes will be picked up automatically. Running a Rails command inside the Avalon container is easy, for example, to run tests ```docker-compose exec avalon bash -c "RAILS_ENV=test bundle exec rspec"```.
```

Comment:
Should this say `puma` instead of `webrick`?

Reply:
✅